wgengine/magicsock/endpoint.go

```diff
@@ -960,22 +962,35 @@ func (de *endpoint) send(buffs [][]byte) error {
 			de.noteBadEndpoint(udpAddr)
 		}
 
+		var txBytes int
+		for _, b := range buffs {
+			txBytes += len(b)
+		}
+
+		switch {
```

Comment:
Perhaps this would be easier to read if you constructed pathLabel once (in a `switch` or just an `if/else` block), and then incremented metrics once after that.